packages/fxa-payments-server/src/lib/apiClient.test.ts

```diff
@@ -1,13 +1,49 @@
+import nock from 'nock';
```

Comment:
I'm also not *super* happy at the repetition in these new apiClient tests - but now at least every API call has a test or two.